src/App/screens/Instructor/screens/GetPublished/components/GetPublishedSteps/utils/isStepComplete.js

```diff
@@ -0,0 +1,16 @@
+import includes from 'lodash/includes'
```

Comment:
I like this a lot. breaking the utilities out into well tested functions that can be imported and used as needed.